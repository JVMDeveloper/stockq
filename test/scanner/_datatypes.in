int
float
bool
void
true
false
struct
array
string
