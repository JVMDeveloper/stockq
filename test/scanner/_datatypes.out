INT
FLOAT
BOOL
VOID
TRUE
FALSE
STRUCT
ARRAY
STRING

